src/obis-observer/utils/message.ts

```diff
@@ -0,0 +1,21 @@
+import {TCommand} from './command.js';
+import {errorResponse} from '../constants/uplinkIds.js';
+import {TCommandId} from '../../types.js';
+
+
+export const findCommand = ( commands: Array<TCommand>, commandId: TCommandId ) => commands.find(command => {
+    if ( 'id' in command ) {
+        return command.id === commandId;
+    }
+
+    if ( 'command' in command ) {
+        return command.command.id === commandId;
+    }
+
+    return null;
+});
+
+export const findCommandError = ( commands: Array<TCommand>, commandId: TCommandId ) => commands.find(command => 'id' in command
+    && command.id === errorResponse
```

Comment:
```suggestion
    && command?.id === errorResponse
```

and then `'id' in command` can be removed